.github/workflows/ci.yaml

```diff
@@ -227,15 +227,11 @@ jobs:
     name: MySQL 5.7 integration test
     runs-on: ubuntu-latest
 
-    services:
-      mysql:
-        image: mysql:5.7
-        env:
-          MYSQL_ROOT_PASSWORD: mysql
-        ports:
-          - 3306:3306
-
     steps:
+      - name: Run MySQL 5.7
+        run: |
+          docker run -e MYSQL_ROOT_PASSWORD=mysql -p 3306:3306 -d mysql:5.7 --character-set-server=utf8mb4 --collation-server=utf8mb4_bin
```

Comment:
Because we need to run MySQL with the collation-related parameters, we run `docker run` command here.